contrib/packaging/rpm/Makefile

```diff
@@ -1,5 +1,6 @@
 include ../../../Makefile.defs
 
+VERSION := $(shell git tag -l --sort=-v:refname | sed 's/v\([^-]*-g\)/r\1/;s/-/./g' | head -1)
```

Comment:
This should be available from `./VERSION`. Did we change anything that broke it?